Gemfile

```diff
@@ -3,9 +3,9 @@ ruby '2.4.1'
 
 gem 'rails', '5.0.3'
 
-gem 'mongo', '2.3.1'
-gem 'bson', '4.1.1'
-gem 'mongoid', '6.0.0'
+gem 'mongo'
+gem 'bson'
+gem 'mongoid'
```

Comment:
We have the above frozen to ensure compat with the production version of MongoDB. I assume all of your tests pass with most recent gems? Which version of MongoDB are you testing against?

Reply:
I removed the version because every things is locked inside Gemfile.lock. so every other dev, and your production website will install the Gemfile.lock during the bundle install

Reply:
And about the version of mongodb, i work with 3.2 but your travis test it on 2.* (Can't remember) so you can be sur this code works on your database 's production version (if you have well configuree teavis.yaml)

Reply:
I suspect we might run into some issues with the new queries when we run them against old Mongo. That said, the fact that the tests are working a-OK on your end with latest version gives me good confidence that we should do the upgrade. It'll be a tricky deploy.. but we need to do it.

Reply:
I was totally wrong about mongo : 
I said  " i work with 3.2"

No. I work with `MongoDB shell version v3.6.4`
And these query doesn't work under 3.4

You where right about "I suspect we might run into some issues with the new queries when we run them against old Mongo"

And I said "your travis test it on 2.*"
Because I read in .travis.yml `- mongodb-2.6-precise`
But if I read raw log at https://api.travis-ci.org/v3/job/379148525/log.txt
we can see : 
```
MongoDB version
MongoDB 3.4.10
```

That s why this commit worked on Travis

Reply:
Figures. No worries, I had a strong suspicion this will be the case anyway. We need to bite the bullet and do the upgrade, which is well overdue.

Full speed ahead.